README.md

```diff
@@ -1,96 +1,179 @@
+* 1 IoT Hub (S1 – Basic tier)
+* 1 Cosmos DB account (Standard)
+* 1 Storage account (Standard -GRS)
+* 1 Web Application
+
+### Basic
+Basic deployment is geared toward showcasing the solution. To reduce the cost
+of this demonstration, all of the microservices are deployed in a single
+virtual machine; this is not considered a production-ready architecture.
+ 
+Our Standard deployment option should be used when you are ready to customize
+a production-ready architecture, built for scale and extensibility.
+ 
+Creating a Basic solution will result in the following Azure services being
+provisioned into your Azure subscription at cost: 
+* 1 Azure Active Directory application
+* 1 Virtual Machine (Standard D1 V2 (1 core, 3.5GB memory))
+* 1 IoT Hub (S1 – Basic tier)
+* 1 Cosmos DB account (Standard)
+* 1 Storage account (Standard -GRS)
+* 1 Web Application
+
+> Pricing information for these services can be found
+[here](https://azure.microsoft.com/pricing). Usage amounts and billing details
+for your subscription can be found in the
+[Azure Portal](https://portal.azure.com/).
+
+Architecture Overview
+=====================
+<div align="center">
+<img src="https://user-images.githubusercontent.com/3317135/31836797-5a6656a0-b58b-11e7-9b64-f31da8bcd10e.png" width="700" height="auto"/>
+</div>
+
+[Open in draw.io][draw-io-map]
+
+## Microservices & Docker Containers
+Remote Monitoring is the first of our preconfigured solutions to leverage a
+microservices architecture available in both .NET and Java. Microservices have
+emerged as a prevalent pattern to achieve scale and flexibility, without
+compromising development speed. Microservices compartmentalize the code and
+provide well defined interfaces making the solution easier to understand and
+less monolithic. It also further expands options for partners that want to
+extend our current preconfigured solutions to build finished solutions that can
+be monetized.
+
+**Learn more about Docker Containers**
+* [Install Docker](https://docs.docker.com/engine/installation/)
+* [Common Docker Commands for Remote Monitoring](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#common-docker-commands)
+* [Docker Getting Started](https://docs.docker.com/get-started/)
+
+## Components
+* [Remote Monitoring Web UI](https://github.com/Azure/pcs-remote-monitoring-webui)
+* [CLI](https://github.com/Azure/pcs-cli)
+* [IoT Hub manager](https://github.com/Azure/iothub-manager-java)
+* [Authentication](https://github.com/Azure/pcs-auth-dotnet)
+* [Device Simulation](https://github.com/Azure/device-simulation-java)
+* [Telemetry](https://github.com/Azure/device-telemetry-java)
+* [Telemetry Agent](https://github.com/Azure/telemetry-agent-java)
+* [Config](https://github.com/azure/pcs-config-java)
+* [Storage Adapter](https://github.com/azure/pcs-storage-adapter-java)
+* [Application Gateway (SSL Proxy WebApp)](https://github.com/Azure/reverse-proxy-dotnet)
```

Comment:
there is also an "API Gateway", currently hosted [here](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/tree/master/reverse-proxy) (we'll have to move it to a dedicated repo)